backend/src/main/java/feedzupzup/backend/admin/domain/AdminRepository.java

```diff
@@ -12,4 +13,17 @@ public interface AdminRepository extends JpaRepository<Admin, Long> {
     Optional<Admin> findByLoginId(final LoginId loginId);
     
     boolean existsByLoginId(final LoginId loginId);
+
+
+    @Query("""
+    SELECT EXISTS (
+      SELECT 1
+      FROM Admin a
+      INNER JOIN Organizer org ON a.id = org.admin.id
+      INNER JOIN Organization o ON o.id = org.organization.id
+      INNER JOIN Feedback f ON f.organization.id = o.id
+      WHERE a.id = :adminId AND f.id = :feedbackId
+    )
+    """)
```

Comment:
```suggestion
  @Query("""
      SELECT EXISTS (
        SELECT 1
        FROM Organizer org
        INNER JOIN Feedback f ON f.organization.id = org.organization.id  
        WHERE org.admin.id = :adminId AND f.id = :feedbackId
      )
      """)
```
p1) 기존 쿼리에는 Admin, Organizer, Organization, Feedback 4개의 테이블이 사용되고 있는데요. 위와 같이  Organizer, Feedback 2개의 테이블만 사용해도 피드백글에 대한 권한이 있는 Admin인지 검증할 수 있을것 같아요~

Reply:
좋슴다!